### DISCUSSION TITLE
HTML for theme switching example? Also, maybe a color selector component?

### DISCUSSION THREAD

**Discussion by GregJohnStewart** [twbs/bootstrap#38691] [Q&A]
While there is a n example bit of js for a color theme picker, there isn't anything for the associated HTML. What should it be?

**Comment by julien-deramond** [comment]
[Color modes](https://getbootstrap.com/docs/5.3/customize/color-modes/#dark-mode) documentation mentions that
> Color modes can be toggled globally on the `<html>` element, or on specific components and elements, thanks to the `data-bs-theme` attribute

So if you are using a color picker, I imagine that you'll switch between light and dark mode. In this case, you can just add `data-bs-theme="auto"` on the `<html>` element. The JS color modes script example will update this value based on user actions and/or configuration.

FYI, we provide a [color modes example](https://github.com/twbs/examples/tree/main/color-modes), and the corresponding [index.html](https://github.com/twbs/examples/tree/main/color-modes/index.html) if it can help.

**Reply by GregJohnStewart** [reply]
That example helps but it doesn't seem to have the associated js? Doesn't seem to work with the js on the docs. Also not as helpful when you don't have npm

And I can understand that it's a simple switch in the `html`, but just odd to me to have the js but not the html example right in the docs

**Reply by GregJohnStewart** [reply]
Got it working, though will note that the js on the bootstrap docs isn't directly compatible with the html in the example
